sw/device/lib/testing/flash_ctrl_testutils.c

```diff
@@ -286,26 +287,54 @@ bool flash_ctrl_testutils_bank_erase(dif_flash_ctrl_state_t *flash_state,
   return retval;
 }
 
-uint32_t flash_ctrl_testutils_get_count(uint32_t *strike_counter) {
-  uint32_t addr = (uint32_t)strike_counter;
-  uint32_t val = abs_mmio_read32(addr);
-
-  for (size_t i = 0; i < 32; ++i) {
-    if (val >> i & 0x1) {
-      return i;
+enum {
+  kNonVolatileCounterFlashWords = 256,
+};
+static_assert(
+    FLASH_CTRL_PARAM_BYTES_PER_WORD == sizeof(uint64_t),
+    "Elements of the counter array must be the same size as a flash word");
+extern const char _non_volatile_counter_flash_words[];
```

Comment:
Why is this `const`?

Reply:
Hmm, doesn't have to be I guess. It looks like we have both patterns in the code base for referencing a symbol defined in a linker script: `extern const char` and `extern char`, the latter being more popular. Do you know if this would affect the generated code? Should we fix the rest too in a separate PR?